spec/Section 7 -- Response.md

```diff
@@ -10,71 +10,75 @@ the case that any _field error_ was raised on a field and was replaced with
 
 ## Response Format
 
-A GraphQL request returns either a _response_ or a _response stream_.
+:: A GraphQL _response_ is the returned result of a GraphQL _request_. A
+_response_ is either a _response payload_ or a _response stream_.
 
-### Response
+### Response Payload
 
-:: A GraphQL request returns a _response_ when the GraphQL operation is a query
-or mutation. A _response_ must be a map.
+:: A GraphQL request returns a _response payload_ when the GraphQL operation is
+a query or mutation. When the GraphQL operation is a subscription, the GraphQL
+request returns a _response payload_ if a _request error_ is raised. A response
+payload must be a map.
 
-If the request raised any errors, the response map must contain an entry with
-key `errors`. The value of this entry is described in the "Errors" section. If
-the request completed without raising any errors, this entry must not be
+If the request raised any errors, the _response payload_ must contain an entry
+with key `errors`. The value of this entry is described in the "Errors" section.
+If the request completed without raising any errors, this entry must not be
 present.
 
-If the request included execution, the response map must contain an entry with
-key `data`. The value of this entry is described in the "Data" section. If the
-request failed before execution, due to a syntax error, missing information, or
-validation error, this entry must not be present.
+If the request included execution, the _response payload_ must contain an entry
+with key `data`. The value of this entry is described in the "Data" section. If
+the request failed before execution, due to a syntax error, missing information,
+or validation error, this entry must not be present.
 
-The response map may also contain an entry with key `extensions`. This entry, if
-set, must have a map as its value. This entry is reserved for implementers to
-extend the protocol however they see fit, and hence there are no additional
-restrictions on its contents.
+The _response payload_ may also contain an entry with key `extensions`. This
+entry, if set, must have a map as its value. This entry is reserved for
+implementers to extend the protocol however they see fit, and hence there are no
+additional restrictions on its contents.
 
 To ensure future changes to the protocol do not break existing services and
-clients, the top level response map must not contain any entries other than the
-three described above.
+clients, the top level _response payload_ must not contain any entries other
+than the three described above.
 
-Note: When `errors` is present in the response, it may be helpful for it to
-appear first when serialized to make it more clear when errors are present in a
-response during debugging.
+Note: When `errors` is present in the _response payload_, it may be helpful for
+it to appear first when serialized to make it more clear when errors are present
+in a response payload during debugging.
```

Comment:
```suggestion
Note: When `errors` is present in the _response payload_, it may be helpful for
it to appear first when serialized to make it more clear that errors are present.
```

nitpick, I thought the repetition was a bit odd, feel free to ignore